virtualization-and-containers/material/exercise_vm_containers_text.md

```diff
@@ -0,0 +1,175 @@
+# Working with Virtualization and Containers
+
+**This exercise sheet is work in progress**
+
+In this exercise we will work the virtualization and container techniques that we have seen in the lecture. We will set up virtual machines manually first and then automatize the process. Afterwards we will build out own containers.
+
+## Deadline
+
+Please finish the mandatory parts of the work before **18 November 2021 at 9:00**.
+
+## Prerequisites
+
+- Installed and working VirtualBox, Vagrant and Docker.
+- GitLab account.
+- Some available space on your hard drive or a USB drive (about 8 GB to be safe).
+
+## Before You Start
+
+- Whenever `USERNAME` is mentioned in a code block or similar, you have to replace this by your GitLab username. For example, if a task states that you shoud use "[`USERNAME`] Ubuntu Server Installation", then you have to actually use "[`jaustar`] Ubuntu Server Installation" in case `jaustar` is the username.
+- We try to check Vagrant and Docker submissions automatically using CI tools to give a better feeling for normal development workflows. This is an experimental feature and might not work perfectly. If you find some mistake, please add an issue in the corresponding repository and assign it to `jaustar`.
+- If you are using a MacBook with ARM CPU and Parallels instead of VirtualBox, please mention this in the issues and pull requests that you create during this exercise. Ideally, use the `Parallels` label on GitLab for that.
+- If you run into problems, check out the "Further Information" sections at the bottom of the tasks. These sections contain information about common pitfalls and link to the documentation of the used tools.
+
+## Virtual Machines Using VirtualBox
+
+We will install "Ubuntu Server 20.04" a lightweight version of Ubuntu without graphical user interface. However, the ISO image has still 1.2 GB so you might want to start the [download of the image called "Ubuntu Server 20.04.3 LTS"](https://ubuntu.com/download/server) before continuing reading. If you see the name "Focal Fossa" some places, this is the codename of the 20.04 release of Ubuntu.
```

Comment:
```suggestion
We will install "Ubuntu Server 20.04", a lightweight version of Ubuntu without graphical user interface. However, the ISO image has still 1.2 GB so you might want to start the [download of the image called "Ubuntu Server 20.04.3 LTS"](https://ubuntu.com/download/server) before continuing reading. If you see the name "Focal Fossa" some places, this is the codename of the 20.04 release of Ubuntu.
```
Actually, the "download task" is just the next one below. You could skip the complete point here.

Reply:
I take the change into the repository and slim it down during the rewriting.